AccountsManager/client/src/main/java/accounts/pojos/AccountData.java

```diff
@@ -0,0 +1,47 @@
+package accounts.pojos;
+
+import org.apache.commons.lang3.RandomStringUtils;
+
+public class AccountData {
+
+    private String name;
+    private String indexName;
+    private long id;
+    private String token;
+
+    public AccountData() {
+    }
+
+    public AccountData(String accountName) {
+        setAccount(accountName);
+    }
+
+    private void setAccount(String accountName) {
```

Comment:
`setAccount` sounds like a setter (which is not beseder)
Let's just put this logic inside the constructor